### ISSUE TITLE
Is it possible to configure the listener via jboss-cli.sh on startup (docker/k8s)?

### ISSUE THREAD

**Issue by ghenadiibatalski** [aerogear/keycloak-metrics-spi#96]
## Description
I would like to configure the SPI via the jboss-clie.sh on startup of a docker container

## Expected Behavior
Is something like
```
/subsystem=keycloak-server/spi=eventsListener:add()
/subsystem=keycloak-server/spi=eventsListener/provider=metrics-listener:add(enabled=true)
```
meaningful/possible and would it work? How could i enable `eventsEnabled=true`  and `adminEventsEnabled=true` ?

Thank you

**Comment by haloss1** [comment]
Interested in this as well, any example on how we could do this?

Thanks

**Comment by madhumithaJarugu** [comment]
@ghenadiibatalski did you get any solution for above scenario ? If so can you help me here

**Comment by ghenadiibatalski** [comment]
@madhumithaJarugu no, we do this manually yet, but if your setup is not too complicated, you should use ansible for the 
consisntency and reproducability purposes. It is possible to configure the listeners via:

events_listeners aliases: eventsListenerslist / elements=string | The realm events listeners.|
 
we do this already for another instances and it does work:
https://docs.ansible.com/ansible/latest/collections/community/general/keycloak_realm_module.html

Good luck!
